apps/website-25/src/components/homepage/CommunitySection/GovernanceProjects.tsx

```diff
@@ -0,0 +1,101 @@
+import { Card, CTALinkOrButton } from '@bluedot/ui';
+
+interface Project {
+  id: number;
+  title: string;
+  authorName: string;
+  imageSrc: string;
+  ctaUrl: string;
+}
+
+const projects: Project[] = [
+  {
+    id: 1,
+    title: 'Contextual Constitutional AI',
+    authorName: 'Akshat Naik',
+    imageSrc: '/images/project-winners/akshat-naik.png',
+    ctaUrl: '#',
+  },
+  {
+    id: 2,
+    title: 'Safety Haven: Justifying and exploring an antitrust safe haven for AI safety research collaboration',
+    authorName: 'Ella Duus',
+    imageSrc: '/images/project-winners/ella-duus.png',
+    ctaUrl: '#',
+  },
+  {
+    id: 3,
+    title: 'Societal Adaptation to AI Human-Labor Automation',
+    authorName: 'Yuval Rymon',
+    imageSrc: '/images/project-winners/yuval-rymon.png',
+    ctaUrl: '#',
+  },
+  {
+    id: 4,
+    title: 'Are you secretly training AI? Methods for uncovering covert AI training: A framework for feasibility and future research',
+    authorName: 'Naci Cankaya',
+    imageSrc: '/images/project-winners/naci-cankaya.png',
+    ctaUrl: '#',
+  },
+];
+
+const GovernanceProjects = () => {
+  return (
+    <section className="governance-projects p-6 container-lined">
+      <div className="governance-projects__container mx-auto max-w-[1750px]">
+        <div className="governance-projects__header mb-12 flex justify-between">
+          <div className="governance-projects__title-container flex flex-col gap-2">
+            <h2 className="governance-projects__title text-2xl font-[650] text-bluedot-darker">
+              AI Governance Projects
+            </h2>
```

Comment:
we should get font styling OOTB now from [globals.css](https://github.com/bluedotimpact/bluedot/blob/6db3fbadd2a7e24ac6ea5567880305d2a06f7c2d/apps/website-25/src/globals.css#L11)
```suggestion
            <h3 className="governance-projects__title">
              AI Governance Projects
            </h3>
```